ds-caselaw-ingester/lambda_function.py

```diff
@@ -156,7 +156,7 @@ def extract_metadata(tar: tarfile, consignment_reference: str):
     te_metadata_file = None
     decoder = json.decoder.JSONDecoder()
     for member in tar.getmembers():
-        if "metadata.json" in member.name:
+        if "-metadata.json" in member.name:
```

Comment:
I know we discussed this, but are we certain we never care to look for a file without the hyphen?

In particular for v1?

Reply:
As far as I can tell, v1 is always emitted with a long filename like below:
<img width="397" alt="image" src="https://github.com/nationalarchives/ds-caselaw-ingester/assets/85497046/f063faff-6626-4a1d-87d3-da700cd65238">

Reply:
great!